pages/model/[uuid]/new-version.tsx

```diff
@@ -141,6 +142,9 @@ function Upload() {
 
     data.schemaRef = model?.schemaRef
 
+    form.append('buildOptions', JSON.stringify(data.buildOptions))
```

Comment:
This is presumably no longer necessary, now that buildOptions is handled by RJSF.

Reply:
oops missed this! deleted